inlong-sort/sort-flink/sort-flink-v1.13/sort-flink-dependencies-v1.13/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
duplicate version v1.13 since the directory name contains flink version

Reply:
This needs to add a version, switch the core dependency or there is no way to distinguish.